rfs/nostr-blossom/src/client.rs

```diff
@@ -111,7 +111,7 @@ impl BlossomClient {
     where
         T: NostrSigner,
     {
-        let mut url = format!("{}/list/{}", self.base_url, pubkey.to_hex());
+        let mut url = format!("{}list/{}", self.base_url, pubkey.to_hex());
```

Comment:
The same here, with something:

```rust
let path: String = format!("list/{}", pubkey.to_hex());
let mut url: Url = self.base_url.join(&path)?;
```

or 

```rust
let mut url: Url = self.base_url.join("list")?.join(&pubkey.to_hex())?;
```

Below, instead of using the `query_params` with `url.push_str`, we can do:

```rust
let mut query_pairs = url.query_pairs_mut();

if let Some(since) = since {
query_pairs.append_pair("since", since.to_string().as_str());
}

if let Some(until) = until {
query_pairs.append_pair("until", until.to_string().as_str());
}
```